tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -211,26 +209,28 @@ export function makeMatMulPackedVec4Source(
             let inputCol = tileCol;
             mm_Bsub[inputRow][inputCol] = mm_readB(batchB, kStart + inputRow, globalCol);
         }
-        kStart = kStart + tileInner;
+        kStart = kStart + ${tileInner};
         workgroupBarrier();
 
         // Compute acc values for a single thread.
-        for (var k = 0; k < tileInner / innerElementSize; k = k + 1) {
-            let BCached0 = mm_Bsub[k * innerElementSize][tileCol];
-            let BCached1 = mm_Bsub[k * innerElementSize + 1][tileCol];
-            let BCached2 = mm_Bsub[k * innerElementSize + 2][tileCol];
+        for (var k = 0; k < ${tileInner} / ${innerElementSize}; k = k + 1) {
```

Comment:
In fact, I prefer `${tileInner} / ${innerElementSize}`, because I view `${}` in WGSL as the equivalence of macros in C/C++.

Reply:
Exactly, `for (var k = 0; k < 32 / 4; k = k + 1)` is more meaningful than `for (var k = 0; k < 8; k = k + 1)`, in case someone would like to read the running WGSL for some reason.
WDYT, @qjia7

Reply:
Personally, I think it's weird that you directly use `32 / 4` not `8` in shader. And from the whole loop body, we can also easily infer that each iteration will process 4 elements.  And I don't know why we need the exact equivalence between wgsl and js code. Why not directly save this division in shader?

Reply:
Either one is OK for me. This will not add runtime cost as Tint and shader compiler can do the optimization.